content/posts/scientific-python/dev-summit-2/index.md

```diff
@@ -0,0 +1,97 @@
+
+This is why, before the summit, we have planning Zoom calls, where we identify topics of interest, which are turned into issues, which are then fleshed out and discussed prior to the event.
+That way, we hoped to hit the ground running—as we did!
+
+## Topics
+
+You can get a rough idea of what we worked on by browsing the [planning issues](https://github.com/scientific-python/summit-2024/issues/) and the [summit worklog](https://hackmd.io/wsJVTMYdQGG_Zgz7rgxSzw).
+
+Broad topics included [SPECs](https://scientific-python.org/specs/), documentation, tools & bots, [lectures](https://lectures.scientific-python.org/), `scipy.sparse`, telemetry, Array API, and type annotation.
+
+### Documentation
+
+Documentation was a much more popular topic than anticipated!
+
+- The new [mystmd](https://mystmd.org/guide) tooling generated some excitement, and an [experimental port of the NumPy tutorials](https://github.com/numpy/numpy-tutorials/tree/mystjs) was made.
+- Recommendations on consistent use of [backticks](https://github.com/numpy/numpydoc/pull/525) and [monospaced font](https://github.com/pydata/pydata-sphinx-theme/issues/1852) were submitted to numpydoc and pydata-sphinx-theme, respectively.
+
+### SPECs
+
+The Scientific Python Ecosystem Coordination documents (SPECs) aim to improve coordination of technical development across the ecosystem.
+
+Several new SPECs were started:
+
+- [SPEC-?: Dispatching (`spatch`)](https://hackmd.io/yI1iAqekQIq0a4jLS9WPyw)
+- [SPEC-8: Securing The Release Process](https://scientific-python.org/specs/spec-0008/)
+- [SPEC-9: Governance](https://github.com/scientific-python/specs/pull/323)
+- [SPEC-10: Changelog and release documentation](https://github.com/scientific-python/specs/pull/321)
+- [SPEC-12: Formatting mathematical expressions](https://github.com/scientific-python/specs/pull/326)
+- [SPEC-13: Naming conventions](https://github.com/scientific-python/specs/pull/324)
+
+Some existing SPECs were discussed and improved:
+
+- [SPEC-7: Seeding pseudo-random number generation (SPRaNG)](https://scientific-python.org/specs/spec-0007/)
+
+### Tooling
+
+- We created a new tools team to handle the ever-growing [list of tools we maintain](https://tools.scientific-python.org/).
+- Eric added his [circleci-artifacts-redirector-action](https://github.com/scientific-python/circleci-artifacts-redirector-action) to the suite.
+- Matthias brought over his [backport bot](https://github.com/scientific-python/MeeseeksDev) and set up a maintenance team.
+
+### SciPy
+
+Several of the SciPy developers were present, and we used the opportunity to celebrate Dan Schult joining as a core developer.
+Matt and Pamphile did some work on the new distribution infrastructure, Dan worked on sparse (remotely with CJ), and newly-supported [`const` statements were added to Cython code](https://github.com/scipy/scipy/pull/20891) got reviewed and merged.
```

Comment:
This last entry (and new-supported `const` statements ...) has two verbs ('were added' and 'got reviewed and merged').

Reply:
Thanks, fixed!